### DISCUSSION TITLE
스케줄 카드 반응형 스타일

### DISCUSSION THREAD

**Discussion by yiminwook** [yiminwook/liveuta-asis#126] [Ideas]
![image](https://github.com/user-attachments/assets/92e4ea6f-7a6f-4f87-924b-882f67cd52b0)

![image](https://github.com/user-attachments/assets/34303f96-b456-4203-be4b-8a20ea6f08eb)

![image](https://github.com/user-attachments/assets/b0acec00-006a-4f5e-8df8-9fbd870e0742)

지금 로딩 placeholder도 가로가 아니라 세로 카드인데 그냥 유투브 따라갈까?

스크롤이 좀 길어지긴할텐데 썸네일은 더 잘보일듯

@utawakuworker 
@pekochan069

**Comment by utawakuworker** [comment]
유튜브 따라간다는게 유튜브에서 보여주는거처럼 가로가 아닌 세로 카드를 쓰겠다는거지? 근데 이거 그냥 유저가 전환할 수 있게 하면 안되나?

**Comment by yiminwook** [comment]
스토리지는 왠만해서 쓰고싶지 않아서

**Reply by utawakuworker** [reply]
스크롤 너무 길어질거같아서 좀 두렵긴 하네
